src/Impostor.Server/Net/Inner/Objects/InnerPlayerControl.cs

```diff
@@ -910,7 +944,7 @@ private async ValueTask<bool> HandleSetStartCounter(ClientPlayer sender, int seq
         {
             if (!sender.IsHost && startCounter != -1)
             {
-                if (await sender.Client.ReportCheatAsync(RpcCalls.MurderPlayer, "Client tried to set start counter as a non-host"))
+                if (await sender.Client.ReportCheatAsync(RpcCalls.SetStartCounter, CheatCategory.MustBeHost, "Client tried to set start counter as a non-host"))
```

Comment:
I assume this being `RpcCalls.MurderPlayer` before was a typo?

Reply:
I could not find a good reason for it to be `MurderPlayer`, so I corrected it